src/Components/CreateImageWizardV2/steps/Review/ReviewStep.tsx

```diff
@@ -34,6 +79,81 @@ const ReviewStep = ({ release, arch }: ReviewStepPropTypes) => {
         >
           <ImageOutputList release={release} arch={arch} />
         </ExpandableSection>
+        <ExpandableSection
+          toggleContent={'Target environments'}
+          onToggle={(_event, isExpandedTargetEnvs) =>
+            onToggleTargetEnvs(isExpandedTargetEnvs)
+          }
+          isExpanded={isExpandedTargetEnvs}
+          isIndented
+          data-testid="target-environments-expandable"
+        >
+          {environment.aws.selected && environment.aws.authorized && (
+            <TargetEnvAWSList
+              manual={awsManual}
+              accountId={awsAccountId}
+              sourceId={awsSourceId}
+            />
+          )}
+          {environment.gcp.selected && environment.gcp.authorized && (
+            <TargetEnvGCPList
+              accountType={gcpAccountType}
+              accountEmail={gcpAccountEmail}
+              domain={gcpDomain}
+            />
```

Comment:
Can't recall when the "Principal" was added. It's empty both in the original wizard and 2.0, we should probably hide it when "Share image with Red Hat Insights only" is selected.

Reply:
yeah, type email and domain shouldn't be displayed when we share with redhatinsights.